emperor/support_files/js/view.js

```diff
@@ -1097,6 +1104,106 @@ DecompositionView.prototype.toggleLabelVisibility = function() {
   this.needsUpdate = true;
 };
 
+
+/**
+ * Set the emissive attribute of the markers
+ *
+ * @param {Bool} emissive Whether the object should be emissive.
+ * @param {Plottable[]} group An array of plottables for which the emissive
+ * attribute. If this object is not provided, all the plottables in the
+ * view will be have the scale set.
+ */
+DecompositionView.prototype.setEmissive = function(emissive, group) {
+  group = group || this.decomp.plottable;
+
+  if (this.decomp.isArrowType()) {
+    throw new Error('Cannot set emissive attribute of arrows');
+  }
+
+  var i = 0, j = 0;
+
+  if (this.UIState.getProperty('view.usesPointCloud') ||
+      this.UIState.getProperty('view.viewType') === 'parallel-plot') {
+    var emissives = this.markers[0].geometry.attributes.emissive;
+
+    // the emissive attribute is a boolean one
+    emissive = (emissive > 0) * 1;
+
+    if (this.markers[0].isPoints) {
+      for (i = 0; i < group.length; i++) {
```

Comment:
... perhaps related to my previous question, should this for's be `for (var i = 0 ...`

Reply:
Same as above, it's to avoid he linter from complaining about variables being re-declared.